Lib/threading.py

```diff
@@ -241,6 +241,10 @@ def release(self):
     def __exit__(self, t, v, tb):
         self.release()
 
+    def locked(self):
+        """Returns whether or not this object is locked."""
```

Comment:
```suggestion
        """Return whether or not this object is locked."""
```

I'm sorry I missed this one and made you change the C implementation docstring. Let's just remove the 's' and re-use that exact sentence for the C docstring as well (again, sorry). I'll let you do the modifications.